test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -10541,6 +10568,9 @@ def revert_to_defaults(self):
         """
         self.logger.info(self.logprefix +
                          "reverting configuration to defaults")
+        self.server.manager(MGR_CMD_DELETE,
```

Comment:
I do not see any change in manager method to remove a scheduler from it's dictionary.